frontend/packages/dev-console/src/components/project-access/ProjectAccess.tsx

```diff
@@ -23,21 +23,27 @@ import {
 } from './project-access-form-submit-utils';
 import { validationSchema } from './project-access-form-validation-utils';
 import ProjectAccessForm from './ProjectAccessForm';
-import { Verb, UserRoleBinding, Roles, roleBinding } from './project-access-form-utils-types';
+import { Verb, UserRoleBinding, roleBinding } from './project-access-form-utils-types';
 
 export interface ProjectAccessProps {
   formName: string;
   namespace: string;
   roleBindings?: { data: []; loaded: boolean; loadError: {} };
+  roles: Roles;
```

Comment:
Based on my change in the hook, you need to change this to:

```suggestion
  roles: { data: Roles; loaded: boolean };
```

This makes it easier to check if the data are loaded, instead of checking that the data are empty or not.